Lib/compileall.py

```diff
@@ -371,6 +389,10 @@ def main():
     if args.opt_levels is None:
         args.opt_levels = [-1]
 
+    if len(args.opt_levels) == 1 and args.hardlink_dupes:
+        parser.error(("Hardlinking of duplicated bytecode makes sense "
+                      "only for more than one optimization level."))
```

Comment:
Maybe mention the option name for optimization level in the error message.

Reply:
The error looks like this:
```
usage: compileall.py [-h] [-l] [-r RECURSION] [-f] [-q] [-b] [-d DESTDIR] [-s STRIPDIR]
                     [-p PREPENDDIR] [-x REGEXP] [-i FILE] [-j WORKERS]
                     [--invalidation-mode {checked-hash,timestamp,unchecked-hash}]
                     [-o OPT_LEVELS] [-e DIR] [--hardlink-dupes]
                     [FILE|DIR ...]
compileall.py: error: Hardlinking of duplicated bytecode makes sense only for more than one optimization level.
```
So the options are shown right above the error. Do you think it makes sense to repeat them in the error message?

Reply:
The current error message is fine, ignore my comment ;-)